assignments/life_expectancy/main.py

```diff
@@ -47,7 +47,8 @@ def main(i_path, region, o_path): # pragma: no cover
     parser = argparse.ArgumentParser()
     parser.add_argument('-i', help= "Input file path.") 
     parser.add_argument('-o', help = "Output file path.")
-    parser.add_argument("-r", default="PT", help = "Desire region to filter the csv.")
+    parser.add_argument("-r",  type = Region, choices = list(Region), \
```

Comment:
Something that's worth pointing it out is that this parser will accept `main.py -r Region.PT` but it won't accept `main.py -r PT`.

If you want to accept the later, you could create a custom class method, like this:

```python
import enum
import argparse

class MyEnum(enum.Enum):
    A = 1
    B = 2
    C = 3

    @classmethod
    def argtype(cls, s: str):
        try:
            return cls[s]
        except KeyError:
            raise argparse.ArgumentTypeError(
                f"{s!r} is not a valid {cls.__name__}")

parser = argparse.ArgumentParser()
parser.add_argument('--enum', type=MyEnum.argtype , choices=MyEnum, required=True)
```

Reply:
Thanks, will take that in consideration